spec/requests/v0/disability_compensation_in_progress_forms_controller_spec.rb

```diff
@@ -118,41 +113,39 @@
             json_response = JSON.parse(response.body)
             expect(json_response['formData']['startedFormVersion']).to eq('2019')
           end
-
-          # if the user with an IPF was not chosen for Toxic Exposure 1.1 release
-          it 'does not return startedFormVersion' do
-            VCR.use_cassette('lighthouse/veteran_verification/disability_rating/200_response') do
-              get v0_disability_compensation_in_progress_form_url(in_progress_form_lighthouse.form_id), params: nil
-            end
-
-            expect(response).to have_http_status(:ok)
-            json_response = JSON.parse(response.body)
-            expect(json_response['formData']['startedFormVersion']).to eq(nil)
-          end
+          #
+          # # if the user with an IPF was not chosen for Toxic Exposure 1.1 release
+          # it 'does not return startedFormVersion' do
+          #   VCR.use_cassette('lighthouse/veteran_verification/disability_rating/200_response') do
+          #     get v0_disability_compensation_in_progress_form_url(in_progress_form_lighthouse.form_id), params: nil
+          #   end
+          #
+          #   expect(response).to have_http_status(:ok)
+          #   json_response = JSON.parse(response.body)
+          #   expect(json_response['formData']['startedFormVersion']).to eq(nil)
+          # end
```

Comment:
afaik we shouldn't be seeing claims anymore with a nil or empty startedFormVersion. if we have good coverage for both the 2019 and 2022 scenarios we may be able to remove these types of tests.

Reply:
thanks, @christinec-fftc . removed the commented tests and changed some verbiage around